roles/deploy/tasks/main.yml

```diff
@@ -2,6 +2,10 @@
 - name: Initialize
   deploy_helper: path="{{ project_root }}" state=present
 
+- name: Update git remote URL
+  shell: "cd {{ project_source_path }} && git remote set-url origin {{ project_git_repo }} || :"
+  register: git_status
+
```

Comment:
@fullyint I think it's just a matter of preference at this point. I usually prefer smaller, easier to understand tasks and offloading the logic as much as possible to Ansible. I think using the builtin function of Ansible over shell (when possible) improve long term maintainability and readability.